InternshipProject#2/Manager/TicketManager.cs

```diff
@@ -38,7 +38,7 @@ public async Task<TicketCreateResponse> CreateTicketAsync(TicketCreateRequest ne
             ticket.Status = 1;
             _context.Tickets.Add(ticket);
             await _context.SaveChangesAsync();
-            WatchRequest req = new WatchRequest(reporterId, ticket.Id);
+            WatchRequest req = new WatchRequest(reporterId, ticket.Id, false);
```

Comment:
You should not create a WatchRequest to create a watcher. You should just create a new Watcher object.

Reply:
Created the object